python/psautohint/__main__.py

```diff
@@ -1,246 +1,91 @@
 # Copyright 2014 Adobe. All rights reserved.
 
+"""
+Auto-hinting program for PostScript, OpenType/CFF and UFO fonts.
+"""
+
 from __future__ import print_function, absolute_import
 
+import argparse
 import os
+import plistlib
 import re
 import sys
+import textwrap
+
+from fontTools.misc.py23 import open
+
+from psautohint._psautohint import version as PSAUTOHINT_VERSION
+from psautohint.autohint import hintFiles, ACFontError, ACHintError, logMsg
+from psautohint.ufoFont import UFOParseError
+from psautohint.ufoFont import kProcessedGlyphsLayer as PROCD_GLYPHS_LAYER
+
+__version__ = PSAUTOHINT_VERSION
+
 
-from psautohint import _psautohint, autohint, ufoFont
+SUPPORTED_FONT_FORMATS = ('PFA', 'PFB', 'CFF', 'OTF', 'UFO')
```

Comment:
Do we really support Type 1 fonts? AFAIK that support was dropped a while ago, so this and the help lines below are a bit misleading until support for Type 1 is back.

Reply:
Not yet, but we will. `psautohint` must support Type 1 otherwise we'll need to keep `autohint` alive for a lot longer. The Type 1 support in **autohint** was done via **tx**; in `psautohint` it will be accomplished via **fontTools**. We've somewhat discussed it on #32.